README.md

```diff
@@ -320,10 +320,11 @@ First, you will need to install libtransistor. Download the [latest release](htt
 
 You will need libusb development headers installed. On Debian-based distros, these are usually in the `libusb-1.0.0-dev` package.
 
-Clone this repo.
+Clone this repo and clone its submodules.
 
 ```
 $ git clone git@github.com:misson20000/twili.git
```

Comment:
`$ git clone --recurse-modules ...` is preferable in my opinion

Reply:
Didn't even know that existed :)